src/GUI/EFCorePowerTools.Shared/Models/ConfigModel.cs

```diff
@@ -23,13 +24,32 @@ public string ConfigPath
             }
         }
 
+        /// <summary>
+        /// Gets or sets the project path to the eftp.*.config.json file.
+        /// </summary>
+        public string ProjectPath
+        {
+            get => _projectPath;
+            set
+            {
+                if (value == _projectPath) return;
+                _projectPath = value;
+                OnPropertyChanged();
+            }
+        }
+
+
         public string DisplayName
         {
             get
             {
                 if (!string.IsNullOrEmpty(_configPath))
                 {
-                    return Path.GetFileName(_configPath);
+                    //I dont test all use of this class, that is why I test if projectpath is used
```

Comment:
If it is the root config file, return: "efpt.config.json"
For other files, return the full path below the project path.

So the complete UI would look like this:

```
efpt.config.json
Folder1\efpt.config.extra.json
Folder2\efpt.config.extra.json

```